Makefile

```diff
@@ -252,7 +252,7 @@ $Q +$(MAKE) --no-print-directory $(addprefix $(BUILD)/,$(1))
 endef
 
 .PHONY: build
-build: $(BUILD)/fmt ## ensure all packages build
+build: $(BUILD)/lint ## ensure all packages build
```

Comment:
can we run both?

Reply:
it's already running both. lint depends on fmt.